### PR TITLE
👍 add instruction to solve the problem of the robot window not being displayed

### PR DESCRIPTION
**Pull request by byplayer**
In my environment, the time limit is not displayed automatically. It may happen other environments.
So I add an FAQ for that.

Please refer https://github.com/robocup-junior/erebus-document/issues/13 .

### CODE REVIEW

File: content/en/docs/Installation/Linux/_index.md [hunk 1]
```diff
@@ -44,4 +44,8 @@ This is probably due to the low performance of your computer. This may be improv
 1. Open Tools -> Preferences in the top navigation bar.
 2. Open the OpenGL tab
 3. Set the various parameters appropriately.
-![](opengl.png)
\ No newline at end of file
+![](opengl.png)
+
+### Competition Controller does not appear
+After some minutes, if the time limit is not displayed, as in the following image, please check the following [help](/docs/tutorials/getting-started/#competition-controller-does-not-appear).
```
Comment (rrrobo) [inline content/en/docs/Installation/Linux/_index.md:55]: Thank you for your PR. I think your suggestion is valid and I am fine to put this note here.
But can you please rephrase the description here? It does not represent the situation well. This help is to solve the problem of the robot window not being displayed, but not related to the "time limit".

**Review by rrrobo — CHANGES_REQUESTED** [review]

File: content/en/docs/Installation/Linux/_index.md (hunk 1 above)
Comment (byplayer) [reply content/en/docs/Installation/Linux/_index.md:55]: @rrrobo 
Thanks for your review. 
I changed description. Could you check again?

**Comment by rrrobo** [comment]
Thanks for your idea and PR, I have added the FAQ for each installation page.  We appreciate your contributions 🙇

**Comment by rrrobo** [comment]
Close #13

**Comment by rrrobo** [comment]
close #13

**Pull request merged**